JBPM-5058 - conditional community/product documentation

mrietveld opened the pull request:
@psiroky Would you mind reviewing and merging this? 

Thanks!

psiroky commented:
Should we add this to other modules as well (not just jbpm-docs)?

manstis commented on jbpm-docs/pom.xml:
```diff
@@ -80,8 +99,77 @@
+  <profiles>
+
+    <profile>
+      <id>community-docs</id>
```
I don't understand the full PR, but elsewhere we've had a profile for "productized" to differentiate between Community and Product.. would it be an idea to follow the same pattern?

psiroky replied:
Yes, I think that would definitely makes sense. We could I think make the community settings the default (not in a profile) and override what we need in the `productized` profile.

mrietveld replied:
I'll add a productized profile that shows the "product" docs.

mrietveld commented:
Good idea, I'll modify the PR so that the other modules also do this.

mrietveld commented:
Moved to the `kie-docs` pom.xml so that it applies to all modules. Also added a "productized" profile that can be activated (and that then deactivates the "community" profile, which by default is active).

psiroky commented:
Will take a look at this later today.

psiroky commented:
Merged by https://github.com/droolsjbpm/kie-docs/commit/f57c4e481bec9ea2f2d6a312b71b70250669058a

(+ additional commit in -bootstrap: https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/commit/3f5c0c528deedd15f3c4371c9f12f342c6eaf5d6)

psiroky commented:
@mrietveld should we backport this to 6.4.x?
